tests/test_dns_cli.py

```diff
@@ -142,7 +142,10 @@ def test_check_domain(check_aws_conn, check_r53, fakefs):
         "commands.dns_cli.check_response",
         return_value="{}"
 )
-def test_assign_domain(check_aws_conn, check_response):
+@patch(
```

Comment:
We can make this a single line. I thought our `pre-commit` hooks would take care of this.

Reply:
fixed